bblfsh/client.py

```diff
@@ -55,7 +55,7 @@ def _get_contents(contents: Optional[Union[str, bytes]], filename: str) -> str:
 
     def parse(self, filename: str, language: Optional[str]=None,
               contents: Optional[str]=None, mode: Optional[ModeType]=None,
-              timeout: Optional[int]=None) -> ResultContext:
+              timeout: int=60) -> ResultContext:
```

Comment:
so after you removed an `Optional` what `0` means? 
Is it "no timeout - wait forever"?

Reply:
`None` was wait forever in fact. I have tested the `0` and negative values and produces a `DEADLINE_EXCEEDED`:

``` 
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.7/site-packages/bblfsh/client.py", line 84, in parse
    response = self._stub_v2.Parse(request, timeout=timeout)
  File "/usr/lib/python3.7/site-packages/grpc/_channel.py", line 565, in __call__
    return _end_unary_response_blocking(state, call, False, None)
  File "/usr/lib/python3.7/site-packages/grpc/_channel.py", line 467, in _end_unary_response_blocking
    raise _Rendezvous(state, None, None, deadline)
grpc._channel._Rendezvous: <_Rendezvous of RPC that terminated with:
	status = StatusCode.DEADLINE_EXCEEDED
	details = "Deadline Exceeded"
```

Reply:
If you mean if there is still a way of waiting forever, passing `None` to the method should suffice. I am adding a comment to make it clear

Reply:
yeah usually 0 or negative number means - wait forever.
In our case, we had an `Option::None` and every `Some(value)` was just a regular timeout.
So, my point is, if we want to switch from `Optional` to some primitive type, like int, long, then 0 should stand for `infinite`, because most likely does not make sense to pass 0ms timeout.

Reply:
Done!